docs/content/preview/integrations/tools/dbeaver-ycql.md

```diff
@@ -33,6 +33,20 @@ Before you can start using DBeaver with YCQL, you need to perform the following:
 
 - Install [DBeaver Enterprise Edition](https://dbeaver.com/download/enterprise/).
 
+- `ca.crt` — root certificate file. If the YugabyteDB cluster has encryption in transit enabled, your client needs `root.ca` to establish a connection. You need to download or generate it from the YugabyteDB cluster and add it to the Java TrustStore before using it in DBeaver. For more information, see [Generate the root certificate file](../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) for instructions on how to generate this file.
+
+  {{< note title="Add `root.crt` to Java Keystore" >}}
+
+To add the CA certificate to the Java Keystore, you can use the following command:
+
+```bash
+keytool -importcert -alias root-ca -file root.crt -keystore keystore_name.jks -storetype JKS -storepass <replace_with_your_password_here> -noprompt
+```
+
+Replace `root.crt` with the path to the CA certificate file, `keystore_name.jks` with the path to the Java Keystore file and `password_here`for the YugabyteDB keystore password.
```

Comment:
what is a yuabytedb keystore password? when was it set?

Reply:
It should be the keystore password, not the YugabyteDB keystore. We do not maintain that; the user has to maintain or know the password they set when converting the certificate to JKS format by adding it to the keystore.